docs: grammar, spelling, and readability fixes (#3044)
fastify/fastify: docs/LTS.md

   [Node.js LTS policy](https://github.com/nodejs/Release) within the
   LTS period of that given Fastify release line.

> A "month" is to be 30 consecutive days.

Reviewer: ```suggestion
A "month" is defined as 30 consecutive days.
```
